### PR TITLE
support nn.SyncBatchNorm

### PR DESCRIPTION
**Pull request by ShoufaChen**
solve the warning: Cannot find rule for <class
'torch.nn.modules.batchnorm.SyncBatchNorm'>. Treat it as zero Macs and
zero Params.

### CODE REVIEW

**Comment by ShoufaChen** [comment]
And pytorch supports nn.SyncBatchNorm since `1.1.0`

**Comment by Lyken17** [comment]
LGTM. Thx for the PR.
